### PR TITLE
Fixes missing import of dateutil.parser

### PR DESCRIPTION
**Pull request by wmendes-ionos**
This PR fixes the issue described by #2258

### CODE REVIEW

File: patroni/dcs/__init__.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import abc
 import dateutil
+import dateutil.parser
```
Comment (CyberDem0n) [inline patroni/dcs/__init__.py:3]: In fact, we also don't need `import dateutil`.

Comment (CyberDem0n) [inline patroni/dcs/__init__.py:3]:
```suggestion
import dateutil.parser
```

**Comment by FxKu** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Comment by CyberDem0n** [comment]
Merged, thank you @wmendes-ionos!

**Pull request merged**